README.md

```diff
@@ -1447,6 +1447,21 @@ def filter(myDataset: Dataset[Person], name: String): Dataset[Int] =
 
 Note that the `run` method returns a `Dataset` transformed by the Quill query using the SQL engine.
 
+Additionally, note that the queries printed from `run(myQuery)` during compile time escape question marks by doubling them in order to
+be able to substitute liftings properly. They are then returned back to their original form before running.
+```scala
+import org.apache.spark.sql.Dataset
+
+run {
+  liftQuery(myDataset).filter(_.field == "?").map(_.anotherField)
+}
+// This is generated during compile time:
+// SELECT x1.anotherField _1 FROM (?) x1 WHERE x1.field = '??'
```

Comment:
I wonder if we should use something else than `??` since there might be values with that string. Maybe some non-unicode character?

Reply:
It's fine to have `"??"` values, they just turn into `"????"` etc... All question marks are doubled and then un-doubled after variable substitution is done.

Reply:
(it's basically the same way java escapes backslashes)

Reply:
Makes sense. Maybe we could use `\?` so it's clear it's escaping the question marl?

Reply:
So `"?"` becomes `"\?"`, what does "`\?`" become? Should it become `"\?\?"` or `"\\?"` or `"\??"`.

Reply:
Okay, let's do `"?"` becomes `"\?"`, `"\?"` becomes `"\\?"` etc... That should be clearest to the user.

Reply:
Ready to go

Reply:
Thank you!! I think the README needs to be updated since the example still uses `??`

Reply:
Fixed the readme